admin/tabs/activity/clienterrors/ClientErrorModel.js

```diff
@@ -53,13 +101,33 @@ export class ClientErrorModel {
     }
 
     async doLoadAsync(loadSpec) {
-        return XH.fetchJson({
-            url: 'clientErrorAdmin',
-            params: this.getParams(),
-            loadSpec
-        }).then(data => {
-            this.gridModel.loadData(data);
-        }).catchDefault();
+        try {
+            const data = await XH.fetchJson({
+                url: 'clientErrorAdmin',
+                params: this.getParams(),
+                loadSpec
+            });
+
+            data.forEach(it => {
+                it.id = `Entry: ${it.id}`;
+                it.day = LocalDate.from(it.dateCreated);
```

Comment:
This should be done on the server, so we have the actual day the user visited the site, not the day as translated to this users timezone!     Can add a LocalDate to the serialization of ClientErrorAdminController and FeedbackAdminController

Reply:
Perhaps we could make the code read here:

it.day = it.day ?: LocalDate.from(it.dateCreated).  

That way when we fix hoist core, it will just start working and we won't have a co-dependency.